packages/medusa/src/commands/exec.ts

```diff
@@ -0,0 +1,42 @@
+import loaders from "../loaders"
+import express from "express"
+import path from "path"
+import logger from "../loaders/logger"
+import { ExecArgs } from "@medusajs/types"
+
+type Options = {
+  file: string
+  args: string[]
+}
+
+export default async function script({ file, args }: Options) {
+  logger.info(`Executing script at ${file}...`)
+  const app = express()
+  const directory = process.cwd()
+
+  try {
+    // set worker mode
+    process.env.MEDUSA_WORKER_MODE = "worker"
+
+    const { container } = await loaders({
+      directory,
+      expressApp: app,
+    })
+
+    const scriptFile = (await import(path.resolve(directory, file))).default
+
```

Comment:
thought(non-blocking): maybe some light validation would be good so we can shoot off a nice error message.

Reply:
Will add it in a separate PR later to not block merging for now 👍🏻